saphanabootstrap-formula.spec

```diff
@@ -19,7 +19,7 @@
 # See also http://en.opensuse.org/openSUSE:Specfile_guidelines
 
 Name:           saphanabootstrap-formula
-Version:        0.4.4
+Version:        0.4.5
```

Comment:
I think this change deserves a bigger version upgrade. 
I would put `0.5.0` as it looks like it's not backward compatible with hanadb_exporter anymore